src/DotLiquid/StandardFilters.cs

```diff
@@ -227,6 +227,26 @@ public static string StripHtml(string input)
         }
 
         /// <summary>
+        /// Converts the input object into a formatted currency as specified by the culture info.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <param name="cultureInfo"></param>
+        /// <returns></returns>
+        public static string Currency(object input, string cultureInfo = "en-US")
+        {
+            decimal amount;
+
+            if (decimal.TryParse(input.ToString(), out amount))
+            {
+                var culture = new CultureInfo(cultureInfo);

```

Comment:
.net Standard 1.3 doesn't have that constructor. Therefore, I changed it to read from the `CurrentCulture` static property